File: Flutter App/unicoVehicle/lib/Models/InsuranceType.dart
import './Miscellaneous/InsuranceCompany.dart';

class InsuranceType {
  final int? insuranceTypeId;
  final String? insuranceType;
  final InsuranceCompany? insuranceCompany;

  InsuranceType({
    this.insuranceCompany,
    this.insuranceType,
    this.insuranceTypeId,
  });
}

File: UnicoVehicle/UnicoVehicle/UnicoVehicle.DTO/InsuranceType.cs
using System;
namespace UnicoVehicle.DTO
{
    public class InsuranceType
    {
        public int InsuranceTypeId { get; set; }
        public string InsuranceTypeName { get; set; }
        public Miscellaneous.InsuranceCompany InsuranceCompany { get; set; }
    }
}
